cmd/api/acrsdk_test.go

```diff
@@ -58,3 +75,158 @@ func TestGetExpiration(t *testing.T) {
 		t.Fatal("Expected error while parsing token, got nil")
 	}
 }
+
+func TestGetAcrCLIClientWithAuth(t *testing.T) {
+	var testLoginURL string
+	testTokenScope := "registry:catalog:* repository:*:*"
+	testAccessToken := strings.Join([]string{
+		base64.RawURLEncoding.EncodeToString([]byte(`{"alg":"RS256"}`)),
+		base64.RawURLEncoding.EncodeToString([]byte(`{"exp":1563910981}`)),
+		"",
+	}, ".")
+	testRefreshToken := "test/refresh/token"
+
+	// create an authorization server
+	as := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		path := r.URL.Path
+		if r.Method != http.MethodPost {
+			w.WriteHeader(http.StatusNotFound)
+			panic("unexpected access")
```

Comment:
Instead of `panic` or simply return 401, you can invoke `t.Fatal` or `t.Error` for proper validation.

Reply:
Replaced all three panic with `t.Fatal/f`